sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentPageLengthUnit.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentPageLengthUnit. */
 @Immutable
 public final class DocumentPageLengthUnit extends ExpandableStringEnum<DocumentPageLengthUnit> {
+
+    /**
+     * Creates a DocumentPageLengthUnit object.
+     */
+    public DocumentPageLengthUnit() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.